src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php

```diff
@@ -13,49 +13,67 @@
 
 namespace Sylius\Bundle\AdminBundle\Controller;
 
-use GuzzleHttp\ClientInterface;
+use GuzzleHttp\ClientInterface as DeprecatedClientInterface;
```

Comment:
I think it would make better sense to alias this as GuzzleClientInterface. At least it would make the change reference in the UPGRADE document more obvious.

Reply:
That's a good point. The alias itself is not a big deal IMO, but in the upgrade file, I would be to use FQCN in this case.